package/dnf.spec.in

```diff
@@ -12,7 +12,7 @@
 
 Name:		dnf
 Version:	@DNF_MAJOR@.@DNF_MINOR@.@DNF_PATCH@
-Release:	2%{?dist}
+Release:	2%{?dist}%{?extra_rel}
```

Comment:
It depends on purpose - `2%{?dist}%{?extra_rel}` will always be greater (newer) than `2%{?dist}` while in the opposite case you can achieve both e.g `dist=.fc21 & extra_ver=012356` =>  2.0123456.fc21 > 2.fc21, while `dist=.fc21 & extra_ver=abcdef` => 2.abcdef.fc21 < 2.fc21.
So I prefer the save variant with extra_ver at the end.